examples/external-api-request/sign_manifest/pyproject.toml

```diff
@@ -0,0 +1,18 @@
+[tool.poetry]
+name = "sign-manifest"
+version = "0.1.0"
+description = ""
+authors = ["GolemFactory <contact@golem.network>"]
+license = "GPL v3"
+readme = "README.md"
+packages = [{include = "sign_manifest"}]
```

Comment:
It is useful to point out to user how they can sign `manifest.json` in a build process, but I always thought we will publish `external-api-request` with a signature under source control.
The goal of the example will be to show working example of an outbound network request. In order to be able to run it we need providers with Golem certificate. In order to sign example's `manifest.json` we would need to share with user one of Golem's private keys. We do not plan to commit private keys into source control.